Sample Data/Custom/GCP_MONITORING_CL.json

```diff
@@ -0,0 +1,272 @@
+[
+    {
+        "TenantId": "00000000-0000-0000-0000-000000000000",
+        "SourceSystem": "RestAPI",
+        "MG": "",
+        "ManagementGroupName": "",
+        "TimeGenerated": "6/16/2021, 2:30:01.271 PM",
+        "Computer": "",
+        "RawData": "",
+        "metric_labels_loadbalanced_s": "",
+        "value_doubleValue_d": "0.00152874941890104",
+        "metric_labels_device_type_s": "",
+        "metric_labels_instance_name_s": "centosps",
+        "metric_labels_device_name_s": "",
+        "metric_labels_storage_type_s": "",
+        "metric_type_s": "compute.googleapis.com/instance/cpu/utilization",
+        "resource_type_s": "gce_instance",
+        "resource_labels_project_id_s": "sometestproject-111111",
+        "resource_labels_instance_id_s": "1111111111111111111",
+        "resource_labels_zone_s": "europe-west3-c",
+        "metricKind_s": "GAUGE",
+        "valueType_s": "DOUBLE",
+        "interval_startTime_t": "6/16/2021, 2:27:00.000 PM",
+        "interval_endTime_t": "6/16/2021, 2:27:00.000 PM",
+        "value_int64Value_d": "",
```

Comment:
please update "value_int64Value_d"  field value to double type as it is ingesting string in LAW. e.g 10, 47934

Reply:
updated